salt/modules/file.py

```diff
@@ -4453,6 +4453,9 @@ def check_perms(name, ret, user, group, mode, attrs=None, follow_symlinks=False)
                 group = perms['lgroup']
             try:
                 chown_func(name, user, group)
+                # Python os.chown() does rest the suid and sgid,
```

Comment:
Typo: `rest` = `reset`?

Reply:
Yes, it is :D 
I will fix it, thanks @rrroo :+1:

Reply:
Fixed.